app/Http/Controllers/Bible/BibleFilesetsDownloadController.php

```diff
@@ -184,15 +184,7 @@ public function list($cache_key = 'bible_filesets_download_list')
             $cache_params,
             now()->addHours(12),
             function () use ($key, $limit, $type) {
-                return BibleFilesetLookup::contentAvailable($key)
-                    ->select(['filesetid', 'type', 'language', 'licensor'])
-                    ->when($type, function ($query) use ($type) {
-                        $set_type_code_array = BibleFileset::getsetTypeCodeFromMedia($type);
-                        $query->whereIn('type', $set_type_code_array);
-                    })
-                    ->distinct()
-                    ->orderBy('filesetid')
```

Comment:
@bradflood The `orderBy` clause has been removed because it is affecting the performance.